docs/deploy.md

```diff
@@ -1,7 +1,6 @@
 # Prerequisite
 
 - [Terraform](https://www.terraform.io/downloads) (>= 0.14)
-- [RKE](https://rancher.com/docs/rke/latest/en/installation/) (>= v1.4.1)
```

Comment:
Why you remove rke ?

Reply:
Not mandatory anymore as rke binary is downloaded